filters/HagDelaunayFilter.hpp

```diff
@@ -0,0 +1,69 @@
+/******************************************************************************
+* Copyright (c) 2016, Bradley J Chambers (brad.chambers@gmail.com)
```

Comment:
I know it's a nit, but I had very little to do with the Delaunay method (if anything at all). Maybe it would be better to include James McClain (2019) for these files.

Reply:
I'm just going to remove it.  I don't know why we do this or how any individual can really claim copyright on this.  But I'm not an IP attorney, thank goodness.